tests/login.test.js

```diff
@@ -0,0 +1,40 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import loginPage from '../pages/login.page';
+import homePage from '../pages/home.page';
+import { expect } from 'chai';
+
+describe('OpenBank', () => {
+
+    
+    it('Should access the home page when attempting to login with valid credentials', async () => {
+        await loginPage.open('login');
+        let username = 'jsmith@demo.io';
+        let password = 'Demo123!';
+        await loginPage.enter(username, password);
+       
+        await assert.equal(await $('h1').getText(),'Dashboard','Error: it did not show the expected text');
+        //expect(await homePage.getDashboardTitle()).to.equal('Dashboard');
```

Comment:
This looks like a proper implementation of this assertion. Why is it commented?

Reply:
cause it wasn't working properly but it's fixed now!